### DISCUSSION TITLE
Student developer pack

### DISCUSSION THREAD

**Discussion by 1diti** [community/community#32441] [GitHub Education]
My request for github student developer pack got accepted but still haven't received any mail.

**Comment by garydsong** [comment]
Looks like your github account is marked as PRO. So I believe it's safe to assume that your Student Developer Pack is active!

**Comment by diamant3** [comment]
if you have specific problem, you can consult to [Github Support](support.github.com) to help you.

**Comment by KalagaHemani** [comment]
Same issue. Its been 27 days that u have raised this question. May I know that did you find any way for it ? And did u get the student developer pack ?
